task1/js/app.js

```diff
@@ -0,0 +1,48 @@
+const statistics = {
+    'total': 0,
+    'strong': 0,
+    'mad': 0,
+    'small': 0,
+    'dead': 0
+};
+
+for (let i = 0; i < zombiesData.length; i++) {
+    if (zombiesData[i] !== undefined) {
+        if (zombiesData[i] < 1 || isNaN(zombiesData[i])) {
+            statistics.dead += 1;
+        } else if (zombiesData[i] >= 1 && zombiesData[i] < 11) {
+            statistics.small += 1;
+        } else if (zombiesData[i] >= 11 && zombiesData[i] < 21) {
+            statistics.mad += 1;
+        } else if (zombiesData[i] >= 21) {
+            statistics.strong += 1;
+        }
+
+        statistics.total += 1;
+    }
+}
+
+const totalElement = document.querySelector('.total-zombies');
+const deadElement = document.querySelector('.dead-zombies');
+const smallElement = document.querySelector('.small-zombies');
+const madElement = document.querySelector('.mad-zombies');
+const strongElement = document.querySelector('.strong-zombies');
+
+const printInConsole = function() {
+    console.log(totalElement.innerText, statistics.total);
+    console.log(deadElement.innerText, statistics.dead);
+    console.log(smallElement.innerText, statistics.small);
+    console.log(madElement.innerText, statistics.mad);
+    console.log(strongElement.innerText, statistics.strong);
+};
+
+const changeHTMLStatistics = function() {
```

Comment:
what's the point of this function declaration?

Reply:
removed